t/plugin/request-validation.t

```diff
@@ -271,46 +231,6 @@ hello1 world
                             "type": "roundrobin"
                         },
                         "uri": "/opentracing"
-                }]],
```

Comment:
@fukiki may I ask the reason to to remove these contents? Thanks.

Reply:
> @fukiki may I ask the reason to to remove these contents? Thanks.

In my test environment, the parameter string is larger than the remain space of the fixed buffer, which causes nginx to report an error. Therefore, the optional parameter was deleted.
The code in `t/plugin/request-validation.t` has been restored.

Reply:
@fukiki Thanks and good to see it's back. 😄